great_expectations/cli/suite.py

```diff
@@ -116,15 +182,52 @@ def suite_new(ctx, expectation_suite, interactive, profile, batch_request, no_ju
         )
         sys.exit(1)
 
-    _suite_new_workflow(
-        context=context,
-        expectation_suite_name=expectation_suite,
-        interactive=interactive,
-        profile=profile,
-        no_jupyter=no_jupyter,
-        usage_event=usage_event_end,
-        batch_request=batch_request,
-    )
+    # TODO: AJB 20210420 add better warning if user passes --non-interactive but either --profile or --batch-request (for this and suite_edit...)
```

Comment:
Should this better warning be added before the release?  If not, we have to create a task for doing so later...

Reply:
Working on it now

Reply:
Done, let me know what you think

Reply:
Lovely!  Thanks!